ui/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,15 @@
+import ls from "localstorage-slim";
```

Comment:
Do we need a lib for this? Why don't we get the value directly from the local storage?

Reply:
This lib was added to avoid having to deal with `JSON.parse` which is a typing hellscape. With this, we can just set `ls.get<T>(VALUE)` to avoid defining all our own parsers and typeguards.

But it looks like this already achieved by get & set `StorageByKey`, so I will look into that.

Reply:
Removed as discussed.